packages/utils/mod.ts

```diff
@@ -146,6 +146,14 @@ export function formatUnitsFromString(price: string, decimal: number) {
   return formatUnits(BigInt(price), decimal);
 }
 
+export function multiplyAndFormatUnits(
```

Comment:
This function seems to be very specific. It looks like it only used in Cart.tsx. So I think it should live there, at least in not-next.